emacs/lsp-cquery.el

```diff
@@ -60,6 +60,12 @@
   :type 'directory
   :group 'cquery)
 
+(defcustom cquery/executable
+  nil
+  "cquery executable filename. Default is $(cquery/root-dir)/build/app"
```

Comment:
cquery/root-dir should just go away - its a weird distinction when we have this, and its otherwise only used for clang-resource-dir, which should just be a separate option defaulting to nil

Reply:
Done